### PR TITLE
[GH-40] Fix cold start deep link events

### PR DESCRIPTION
**Pull request by rlepinski**

### CODE REVIEW

File: ios/UARCTModule/UARCTEventEmitter.m [hunk 1]
```diff
@@ -77,25 +77,32 @@ - (BOOL)isObserving {
...
+        [self.pendingEvents addObject:@{ @"name": UARCTDeepLinkEventName, @"body": body}];
     }
 }
 
 #pragma mark -
 #pragma mark UAPushDelegate
 
--(void)receivedForegroundNotification:(UANotificationContent *)notificationContent completionHandler:(void (^)())completionHandler {
+-(void)receivedForegroundNotification:(UANotificationContent *)notificationContent
+                    completionHandler:(void (^)(void))completionHandler {
```
Comment (rlepinski) [inline ios/UARCTModule/UARCTEventEmitter.m]: updated signature with extra void and multilined it

File: ios/UARCTModule/UARCTEventEmitter.m [hunk 2]
```diff
@@ -77,25 +77,32 @@ - (BOOL)isObserving {
...
+
     [self sendEventWithName:UARCTPushReceivedEventName body:[self eventBodyForNotificationContent:notificationContent]];
     completionHandler();
 }
 
--(void)receivedBackgroundNotification:(UANotificationContent *)notificationContent completionHandler:(void (^)(UIBackgroundFetchResult))completionHandler {
+-(void)receivedBackgroundNotification:(UANotificationContent *)notificationContent
```
Comment (rlepinski) [inline ios/UARCTModule/UARCTEventEmitter.m]: updated signature with extra void and multilined it

File: ios/UARCTModule/UARCTEventEmitter.m [hunk 3]
```diff
@@ -77,25 +77,32 @@ - (BOOL)isObserving {
 
 -(void)deepLinkReceived:(NSString *)deepLink {
     id body = @{ @"deepLink" : deepLink };
-    if ([self sendEventWithName:UARCTDeepLinkEventName body:body]) {
-        [self.pendingEvents addObject:@{ @"name": UARCTNotificationResponseEventName, @"body": body}];
+
+    if (![self sendEventWithName:UARCTDeepLinkEventName body:body]) {
+        [self.pendingEvents addObject:@{ @"name": UARCTDeepLinkEventName, @"body": body}];
```
Comment (rlepinski) [inline ios/UARCTModule/UARCTEventEmitter.m]: the real fix ^

**Comment by crow** [comment]
🍔

**Comment by marc-scig** [comment]
🌯

**Pull request merged**